### PR TITLE
U/jrbogart/ellipticities

### PR DESCRIPTION
**Pull request by JoanneBogart**
Handle ellipticities properly to create
* skyCatalog output appropriate for reproducing DC2-like results
* skyCatalog output which treats ellipticities, etc. properly

Also involves switching from storing position angle to storing ellipticities (by galaxy component)

### CODE REVIEW

File: python/desc/skycatalogs/objects/base_object.py [hunk 1]
```diff
@@ -305,7 +309,9 @@ def get_gsobject_components(self, gsparams=None, rng=None):
                 n = round(n*20.)/20.
                 obj = galsim.Sersic(n=n, half_light_radius=hlr,
                                     gsparams=gsparams)
-            shear = galsim.Shear(q=b/a, beta=beta)
+
+
```
Comment (jchiang87) [inline python/desc/skycatalogs/objects/base_object.py:313]: Maybe put a comment here that we still need to understand whether the gcr-catalogs position angle needs that extra +90 degrees, amounting to those minus signs for `e1` and `e2`?   Those minus signs may end up as yet another special case to reproduce DC2 galaxy shapes and orientations.

**Review by jchiang87 — APPROVED** [review]
I have a comment on the ellipticity signs, but otherwise looks good for generating the catalogs.

File: python/desc/skycatalogs/objects/base_object.py (hunk 1 above)
Comment (JoanneBogart) [reply python/desc/skycatalogs/objects/base_object.py]: I've added something.   Please let me know if it's not what you had in mind.

Comment (jchiang87) [reply python/desc/skycatalogs/objects/base_object.py]: Yes, looks good.

**Pull request merged**